.github/actions/check-changes/action.yaml

```diff
@@ -15,12 +15,16 @@ inputs:
     description: Use fallback compare against prior commit
     required: false
     default: 'true'
+  diffFilter:
+    description: git diff-filter string to use
+    required: false
+    default: ''
 outputs:
   hasChangedFiles:
     value: ${{ steps.checks.outputs.hasChangedFiles }}
     description: True if specified files were changed in comparison to specified git reference
   changedFiles:
-    value: ${{ toJSON(steps.checks.outputs.changedFiles) }}
+    value: ${{ steps.checks.outputs.changedFiles }}
```

Comment:
Intentional?

Reply:
See above, the type of outputs is `string` so converting to a complex type will not work.

Reply:
Got it.